exemple11.html

```diff
@@ -36,6 +36,7 @@
 	</style>
 </head>
 <body>
+	<script></script>
```

Comment:
1. extract styles into separated file
2. create file for scripts
3. write new simple script and add it to the project 
4. why you use only #? want to see classes too 
5. You use wery large indentation, use one tab ( or two spaces )